apps/prairielearn/src/models/course.ts

```diff
@@ -0,0 +1,14 @@
+import { loadSqlEquiv, queryRow } from '@prairielearn/postgres';
+import { Course, CourseSchema } from '../lib/db-types';
+
+const sql = loadSqlEquiv(__filename);
+
+export async function selectCourse({ course_id }: { course_id: string }): Promise<Course> {
```

Comment:
Nit: in PT we've been naming these functions something like `selectCourseById` and then taking a single `string` argument.

Reply:
Sounds good to me.

Reply:
I changed this one and the `selectQuestion` one as well.